boards/common/nrf52/Kconfig

```diff
@@ -9,3 +9,31 @@ config BOARD_COMMON_NRF52
     select HAS_PERIPH_RTT
     select HAS_PERIPH_TIMER
     select HAS_RIOTBOOT
+
+if TEST_KCONFIG
+
+config MODULE_SAUL_DEFAULT
+    select MODULE_SAUL_NRF_TEMPERATURE
+
+choice
+    bool "Backend"
+    depends on MODULE_NETDEV_DEFAULT
+    default NRF802154
+    default NRFBLE
+
+config NRF802154
+    bool "nrf802154"
+    select MODULE_NRF802154
+
+config NRFBLE
+    bool "nrfble"
+    select MODULE_NRFBLE
+
+config NRFMIN
+    bool "nrfmin"
+    select MODULE_NRFMIN
+
+endchoice
```

Comment:
I think we should improve documentation of this, just "Backend" seems strange. I'm not sure this actually depends on netdev_default but is activated when that module is there. I think we may be missing some driver switch?

Reply:
>  I think we may be missing some driver switch?

Can you explain more?